tensorflow/python/ops/variable_scope.py

```diff
@@ -882,6 +882,11 @@ def set_custom_getter(self, custom_getter):
     """Set custom getter for this scope."""
     self._custom_getter = custom_getter
 
+  def get_collection(self, name):
```

Comment:
Since this adds a new method to the VariableScope class, it will require API change review. 

Also, assuming that this change is all right, it will require some unit tests.